src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/App.razor

```diff
@@ -1,7 +1,14 @@
-﻿<CascadingAuthenticationState>
-    <Router AppAssembly="typeof(Startup).Assembly">
-        <NotFound>
+﻿<Router AppAssembly="@typeof(Program).Assembly">
+    <Found Context="routeData">
+@*#if (OrganizationalAuth || IndividualAuth)
+        <AuthorizeRouteView RouteData="@routeData" DefaultLayout="@typeof(MainLayout)" />
```

Comment:
I'm **REALLY** happy with the outcome of what you changed since our original design discussion about this. This is much more approachable now that it encapsulates the routing part, and has defaults.